app/styles/_section-list.scss

```diff
@@ -0,0 +1,60 @@
+.bc-c-section-list__items {
+  list-style: none;
+  padding-left: 0;
+  @media (min-width: 641px) {
+    padding-left: 30px;
+  }
+}
+
+.bc-c-inset-text {
+  margin-top: nhsuk-spacing(3);
+  max-width: 100%;
+  padding: 0;
+  .bc-c-section-list__item:first-child {
+    border-top: none;
+  }
+  .bc-c-section-list__items {
+    padding-left: 0;
+  }
+}
+
+.bc-c-section-list__item {
+  padding-top: nhsuk-spacing(2);
+  padding-bottom: nhsuk-spacing(2);
+}
+
+.bc-c-section-list__item:after {
+    content: "";
+    display: block;
+    clear: both;
+}
+
+.bc-c-section-list__task-default-message {
+  display: block;
+  margin-left: nhsuk-spacing(3);
+  @media (min-width: 641px) {
+    float: left;
+    margin-left: nhsuk-spacing(1);
+  }
+}
+
+// bc-c-tag is also in public-browse
+.bc-c-tag {
```

Comment:
If we import the buying-catalogue component styles properly. We shouldn't need this right?

Reply:
depends where the styles live - at the moment these ones live in public-browse (I wrote the comment to remind myself that we need to discuss this)

Reply:
I think personally this should live in the components-libarary since it is specific to that component